packages/components/psammead-story-promo/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.3.5   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```

Comment:
```suggestion
| 2.3.3   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```

Reply:
Where has the `2.3.3` gone? 🙈

Reply:
![](https://media.giphy.com/media/L17Rt82QrScQU/giphy.gif)